pandas/io/html.py

```diff
@@ -127,8 +127,8 @@ def _read(obj):
     raw_text : str
     """
     if _is_url(obj):
-        with urlopen(obj) as url:
```

Comment:
@WillAyd in my PR all `urlopen` calls are wrap with `requests` if it is available so I guess we cover this one in #21504